src/dawn_native/Device.cpp

```diff
@@ -218,6 +224,13 @@ namespace dawn_native {
         return {};
     }
 
+    MaybeError DeviceBase::CreateTextureInternal(TextureBase** result,
```

Comment:
ultra-nit: can this be sorted among the CreateFooInternal function?

Reply:
Done